ThomasShelby.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class ThomasShelby {
+    static final int MAX_TASKS = 100;
+
+    static Task[] taskManager = new Task[MAX_TASKS];
+    static int taskCount = 0;
+
+    private static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println((i + 1) + ". "
+                    + taskManager[i].getStatusIcon() + " "
+                    + taskManager[i].getDescription());
+        }
+    }
+
+    private static void addTask(String[] cmdSplit) {
+        Task newTask = new Task(cmdSplit[1]);
+        newTask.setDescription(cmdSplit[1]);
+        taskManager[taskCount] = newTask;
+        taskCount++;
+        System.out.println("added: " + cmdSplit[1]);
+    }
+
+    private static void markTask(String[] cmdSplit) {
+        int whichTask;
+        whichTask = Integer.parseInt(cmdSplit[1]) - 1; // shift back index
+        taskManager[whichTask].setIsDone(true);
+        System.out.println("That was long due, well done.");
+        System.out.println(taskManager[whichTask].getStatusIcon()
+                + " " + taskManager[whichTask].getDescription());
```

Comment:
Good work on keeping the line length no longer than 120 characters throughout your code. And indenting by 8 spaces for wrapped lines.